docs/podman.1.md

```diff
@@ -125,6 +125,8 @@ storage.conf is the storage configuration file for all tools using containers/st
 
 The storage configuration file specifies all of the available container storage options for tools using shared container storage.
 
+When Podman runs in rootless mode, the file `$HOME/.config/containers/storage.conf` is also loaded.
```

Comment:
When Podman runs in rootless mode, the file `$HOME/.config/containers/storage.conf` is loaded instead if it exists.